### DISCUSSION TITLE
欢迎来到“fk龙猫校园”项目讨论区！

### DISCUSSION THREAD

**Discussion by BeiyanYunyi** [lixiang810/totoroschool-autorun#2] [General]
## 👋 欢迎！
  我们正在使用 Github Discussions 作为一个地方与我们社区的成员联系。在这里，我们希望您
  * 提出你想知道的问题
  * 分享想法。
  * 与其他社区成员互动。
  * 欢迎他人并保持开放的心态。请记住，这是一个我们的社区
  共建💪。

  要想开始，请在下面留言，介绍自己，并告诉我们你在这个社区做什么。

**Comment by fangyuanone** [comment]
大佬，可以带一下怎么改fk龙猫的跑步次数吗，老实人老是吃亏，白跑

**Comment by fangyuanone** [comment]
TG   https://t.me/Dadinone 大佬可以带带我么

**Comment by willwongprd** [comment]
由同学介绍的fk-totoro-exp在iOS上成功实施，前来膜拜。（似乎并未适配Android？）

**Reply by BeiyanYunyi** [reply]
上学期是可以的，那时候玩得可欢了。可惜现在不行了。

**Reply by willwongprd** [reply]
有机会我再调教试试吧，总之，运用池沼（确信）来艹猫的行为深得人心，更能彰显出totoro反对者的高雅（，非常的ao用

**Comment by VoxHwa** [comment]
龙猫服务器今天早上g了，😂，舒服了

**Comment by Nohesdc** [comment]
这个能不能刷引体向上等室内测试数据啊

**Reply by BeiyanYunyi** [reply]
不是很清楚这一块的功能，我的学校没这个要求，所以我也没得抓包
